sycl/test-e2e/syclcompat/device/device.cpp

```diff
@@ -392,6 +392,18 @@ void test_get_device_id() {
   assert(id == 0);
 }
 
+void test_event_profiling() {
+  sycl::queue q = syclcompat::get_default_queue();
+
+  if (!q.get_device().has(sycl::aspect::queue_profiling)) {
+    std::cout << "Device does not have aspect::queue_profiling, skipping."
+              << std::endl;
+    return;
+  }
+
+  assert(!q.has_property<sycl::property::queue::enable_profiling>());
+}
+
```

Comment:
I like that you have both positive & negative testing :+1:  I wonder if there's a way to include them both in the same place? You could redefine the value of `SYCLCOMPAT_PROFILING_ENABLED` or something?

It's just for tidiness though - if it's difficult then no worries.

Reply:
Ah, good idea. I can define two different runs using lit instead of defining the macro directly in the header. I'll see how it looks and ping again.